index.md

```diff
@@ -14,7 +14,8 @@ excerpt: "The Belgium Azure User Group focuses on knowledge sharing and networki
 
 ## Upcoming events
 
-No events are currently planned. Keep an eye on our newsletter (see below to subscribe).
+**2019-05-08 - Binding your serverless clients** - Want to learn how you can write your own Azure Function bindings? Then this event is for you! [Register here!](https://www.azug.be/events/2019/05/08/Binding-your-serverless-clients){: .btn .btn--success}
```

Comment:
I'm not sure if this URL is correct - Couldn't get this working locally via `bundle exec jekyll serve` as it didn't list the event in the overview

Reply:
Think the URL will be lowercase `B`, but this may just work...

Reply:
Changed it!